source/user-manual/reference/ossec-conf/client.rst

```diff
@@ -233,6 +234,19 @@ For example, a ``notify_time`` setting of 60 combined with a time-reconnect of 3
 .. warning::
 	Notice that the ``notify_time`` value uses an underscore while the ``time-reconnect`` value uses a dash.  This is an unfortunate legacy naming inconsistency that is easy to mix up.
 
+  .. _force_reconnect_interval:
```

Comment:
We need to add information about the version that introduces this option (4.3.0).

Reply:
Fixed